app/routes/header.tsx

```diff
@@ -24,42 +24,79 @@ export function Header() {
           <h2 className="h2">Call dibs on one of our sportsball courts</h2>
           <fetcher.Form method="post" action="/some/route">
             <div className="header_illustration">
-              <div className="header_icon header_icon___pickleball">
-                <img alt="pball" src="/assets/pickleball-solid.svg" />
-              </div>
               {user ? (
-                <input
-                  id="hidePb"
-                  name="hidePb"
-                  type="checkbox"
-                  defaultChecked={!user.courtViz?.hidePb}
-                  onChange={submit}
-                />
-              ) : null}
-              <div className="header_icon header_icon___tennis">
-                <img alt="tennis racquet" src="/assets/tennis-ball-solid.svg" />
-              </div>
+                <label className="checkbox_wrapper">
+                  <input
+                    className="checkbox_input"
+                    id="hidePb"
+                    name="hidePb"
+                    type="checkbox"
+                    defaultChecked={!user.courtViz?.hidePb}
+                    onChange={submit}
+                  />
+                  <div className="header_checkmark">
+                    <img alt="check" src="https://svgshare.com/i/18ir.svg" />
```

Comment:
i'm 95% sure that if you make the change i suggested above, this will work just fine.
```suggestion
                    <img alt="check" src="/assets/check-solid.svg" />
```